opentreemap/treemap/audit.py

```diff
@@ -140,13 +140,13 @@ def approve_or_reject_audits_and_apply(audits, user, approved):
 
 def add_default_permissions(instance, roles=None, models=None):
     # Audit is imported into models, so models can't be imported up top
-    from treemap.models import MapFeature
+    from treemap.models import MapFeaturePhoto
     if roles is None:
         roles = Role.objects.filter(instance=instance)
     if models is None:
-        # MapFeature is "Authorizable", but it is effectively abstract
-        # Only it's subclasses should have permissions added
-        models = all_subclasses(Authorizable) - {MapFeature, PendingAuditable}
+        # MapFeaturePhoto is an exception to the usual pattern - it is
+        # the only class that we use directly *and* inherit.
```

Comment:
Much better. How about this rewritten comment:

We need permissions only on those subclasses of `Authorizable` which we instantiate. Those are the leaf nodes of the subclass tree, plus `MapFeaturePhoto` (which has subclass `TreePhoto`).